### PR TITLE
Update MITRE ATT@CK Information

### PR DESCRIPTION
**Pull request by spsocprime**
Required items, please complete
   
   Change(s):
   - Updated content in mitre.csv
   - Updated data related to MITRE ATT&CK tactics and techniques in Export-AzSentinelMITREToCSV.ps1

   Reason for Change(s):
   - Updates to reflect changes introduced in MITRE ATT&CK v14: new techniques; updated descriptions.

### CODE REVIEW

**Comment by v-prasadboke** [comment]
Hello @spsocprime , Thanks for raising this PR.
This PR will be investigated and will provide you some updates before 17 November, 2023.

File: Tools/validate-detections/mitre.csv
Comment (rahul0216) [inline Tools/validate-detections/mitre.csv:1]: This file is being used to validate Tactics and Techniques mapping in Analytic Rule, which reflected in MITRE ATT&CK Framework blade. But the blade support only till MITRE ATT&CK Frameworkv11.
So revert the changes made to this file. We'll let you, when support for v14 implemented.

Comment (spsocprime) [reply Tools/validate-detections/mitre.csv:1]: Thanks for replying.
Considering current version Sentinel MITRE Framework blade supports, when you implement v14 - the latest version of MITRE matrix will be probably 17 if not higher
